core/src/Cuda/KokkosExp_Cuda_IterateTile_Refactor.hpp

```diff
@@ -1562,7 +1560,6 @@ struct DeviceIterateTile<
             tile_idx /= m_rp.m_tile_end[i];
 
             // tile-local indices identified with (index_type)threadIdx.y
-            m_local_offset[i] = (thrd_idx % m_rp.m_tile[i]);
```

Comment:
Did you remove this line intentionally?

Reply:
no. What?? That wasn't in the `git commit -p`...